rcp/org.testeditor.rcp4.views.teststepselection/src/org/testeditor/rcp4/views/teststepselector/TestStepSelector.xtend

```diff
@@ -93,37 +113,49 @@ class TestStepSelector {
 
 	@Inject
 	@Optional
-	def void updateModelAndView(@EventTopic(SELECTOR_UPDATE_MODEL_AND_VIEW) Object data) {
-		logger.debug("updateModelAndView")
-		val amlInjector = amlInjectorProvider.get
-		models = amlInjector.getInstance(AmlModelsProvider).amlModels
-		Display.getDefault.syncExec[internalUpdateView]
+	def void updateView(@EventTopic(SELECTOR_UPDATE_VIEW) Object data) {
+		logger.debug("updateView for data='{}'", data)
```

Comment:
logging that the method is called does not provide as much benefit as logging that the update was actually triggered. So it would be much nicer to log something like "updating view for editor='{}'". Otherwise I read the log and think "oh hey, the update was triggered - I see that line in the log" but wonder why nothing happened - maybe the event data has changed and no longer contains the editor.